src/index.test.js

```diff
@@ -240,7 +231,8 @@ test('reads text from fs if filePath is provided but not text', () => {
   } catch (e) {
     // ignore
   }
-  expect(fsMock.readFileSync).toHaveBeenCalledTimes(1)
+  // Two hits to .eslintignore
```

Comment:
I like to try to be explicit about how my mock functions are called. Could you add assertions to verify that it was called twice for `.eslintignore` and also add a comment to explain why it did that?